src/mocks/handlers.ts

```diff
@@ -3,6 +3,57 @@ import { rest } from "msw"
 export const handlers = [
   rest.get("https://pro-api.coinmarketcap.com/v1/cryptocurrency/quotes/latest", (req, res, ctx) => {
```

Comment:
these are mocks. so that when we run tests rather than calling out the the external api, which would have varying value, it returns this value all the time.